jena-arq/src/main/java/org/apache/jena/query/DatasetFactory.java

```diff
@@ -16,253 +16,226 @@
  * limitations under the License.
  */
 
-package org.apache.jena.query ;
-
```

Comment:
See your comment [here](https://mail-archives.apache.org/mod_mbox/jena-dev/201510.mbox/%3C5610F36D.8090602%40apache.org%3E) about "other clearing up of DatasetFactory ...", although I'm not sure why Git created the diff in this confusing way.